Config.md

```diff
@@ -64,29 +64,67 @@ with nxp_config(n_jobs=7, verbose=0):
+
+**Key Takeaway**: When both systems are used together, NetworkX's configuration (`nx.config.backends.parallel`) takes precedence for `nx-parallel` functions. To avoid unexpected behavior, ensure that the `active` setting aligns with your intended configuration system.
+
+### 3.2 Key Differences
+
+- **Parameter Handling**: The main difference is how `backend_params` are passed. Since, in networkx configurations are stored as a [`@dataclass`](https://docs.python.org/3/library/dataclasses.html), we need to pass them as a dictionary, whereas in `joblib.parallel_config` you can just pass them along with the other configurations, as shown below:
+
+  ```py
+  nx.config.backends.parallel.backend_params = {"max_nbytes": None}
+  joblib.parallel_config(backend="loky", max_nbytes=None)
+  ```
 
-- **Usage**: The main difference is how `backend_params` are passed. In `nx.config` you need to pass them as a dictionary, whereas in `joblib.parallel_config` you can just pass them along with the other configurations.
-- **Default Settings**: By default, `nx-parallel` looks for configs set by `joblib.parallel_config`, and NetworkX's config system needs to be enabled explicitly by making the `nx.config.backends.parallel.active` configuration `True`.
+- **Default Behavior**: By default, `nx-parallel` looks for configs in `joblib.parallel_config` unless `nx.config.backends.parallel.active` is set to `True`.
 
-### 3.2 When Should You Use Which System?
+### 3.3 When Should You Use Which System?
 
-- Use NetworkX's config system if you're working with multiple backends within NetworkX.
-- If you're using `nx-parallel` independently, you can choose either system based on your preference.
+- When you're working with multiple NetworkX backends, then to avoid having any conflicts with other backends' code, using networkx to configure nx-parallel would be well suited.
+- If using `nx-parallel` independently, either system can be used based on your preference.
 
-## 4. Important Notes
+## 4. Important Considerations
 
-- **What if you use both systems together?** If you try to use both config systems, NetworkX's configuration will take priority, and `joblib`'s settings will be ignored. To avoid this, make sure `networkx.config.backends.parallel.active` is set to `False` when using `joblib` to set configs, and `True` when using `networkx`.
-- **Why are there two config systems?** We're working towards syncing both systems in the future so that updating one will automatically update the other. An attempt to move towards a unified configuration system in nx-parallel was made by the PR#68, which introduced a custom context manager for `nx-parallel`.
+- **Avoiding Conflicts:** To prevent conflicts, ensure that `networkx.config.backends.parallel.active` is set to `False` when using `joblib`, and `True` when using NetworkX configurations.
```

Comment:
Does "when using `joblib`" imply all use of joblib including in nx-parallel and outside nx-parallel? Or does this refer to using `joblib.parallel_config` to set the configuration for nx-parallel?

Reply:
here, "when using joblib" means when the user is using `joblib.parallel_config` to configure nx-parallel's internal `joblib.Parallel()`.